guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -761,6 +761,18 @@ class SampleEventObserverThatModifiesInputs
 
 15.8. Frequently update the third-party libraries used in the project/component to eliminate known vulnerabilities.
 
+15.9. LFI protection.
+
+15.9.1. SHOULD NOT trust user-submitted requests containing path and file name.
+
+15.9.2. SHOULD sanitize user-submitted path and file values to remove dot-dot-slash from the request.
+
+15.10. RCE protection.
+
+15.10.1. SHOULD avoid usage of eval(), passthru(), system(), shell_exec()
```

Comment:
We also do not recommend usage of `unserialize`, `srand`, `mt_srand`, `md5` (see https://github.com/magento/magento2/blob/2.2-develop/dev/tests/static/testsuite/Magento/Test/Legacy/_files/security/unsecure_php_functions.php)

Reply:
We should rephrase 15.10.1  for clarity.  SHOULD NOT use eval(), passthru(), system, shell_exec().

Reply:
thanks @buskamuza for that.
@szurek sure, totally makes sense.